index.bs

```diff
@@ -1387,33 +1388,72 @@ To <dfn>serialize an [=event-level report=]</dfn> |report|, run the following st
     [=serialize an integer|serialized=].
 1. Return the [=byte sequence=] resulting from executing [=serialize an infra value to JSON bytes=] on |data|.
 
+To <dfn>serialize an [=aggregatable report=] </dfn> |report|, run the following steps:
+
+1. Let |destination| be |report|'s [=aggregatable report/attribution destination=].
+1. [=Assert=]: |destination| is not an [=opaque origin=].
```

Comment:
Since we have this assertion for both event-level and aggregatable, can we move it to the general report serialization algorithm?